aacgmv2/wrapper.py

```diff
@@ -12,12 +13,58 @@
 convert_bool_to_bit : Convert boolian flags to bits
 convert_mlt : Get array mlt
 --------------
+
 """
 
 from __future__ import division, absolute_import, unicode_literals
 import datetime as dt
 import numpy as np
-import logbook as logging
+
+def test_height(height, bit_code):
+    """ Test the input height and ensure it is appropriate for the method
+
+    Parameters
+    ----------
+    height : (float)
+        Height to test in km
+    bit_code : (int)
+        Code string denoting method to use
+
+    Returns
+    -------
+    good_height : (boolean)
+        True if height and method are appropriate, False if not
+
+    """
+    import aacgmv2
+    from aacgmv2._aacgmv2 import TRACE, ALLOWTRACE, BADIDEA
+
+    # Test for heights that are allowed but not smart
+    if height < 0:
+        aacgmv2.logger.warning('conversion not intended for altitudes < 0 km')
+
+    # Test the conditions for using the coefficient method
+    if(height > aacgmv2.high_alt_coeff and
+       not (bit_code & (TRACE|ALLOWTRACE|BADIDEA))):
+        estr = ''.join(['coefficients are not valid for altitudes above ',
```

Comment:
If the height isn't good, you can still run the program.  But because the results are suspect, you have to specify that you know it's a bad idea.  

I output to a logfile instead of stdout because a lot of users will run this on a large file.  Some of the heights in the file won't be appropriate and they don't want outputs for them and they also don't want a lot of warning messages filling up the screen.